packages/remark-parse/lib/tokenize/list.js

```diff
@@ -467,7 +468,7 @@ function normalListItem(ctx, value, position) {
       $2 = C_SPACE + $2;
     }
 
-    max = $1 + repeat(C_SPACE, $2.length) + $3;
+    max = $1 + repeat(C_SPACE, Math.max($2.length, TAB_SIZE)) + $3;
```

Comment:
This is necessary because we want to trim ensuing lines if they contain at least a `\t`, but if the first line doeesn't contain a `\t`, we won't properly trim them here.

I've looked through the changed (failing) test cases that this edit introduces, and AFAICT, their output fixtures were less correct before this change.